### PR TITLE
Fixing CNAME creation in Route 53

### PR DESCRIPTION
**Pull request by krishnan-chandra**
@optimizely/data-services

### CODE REVIEW

File: providers/record.rb [hunk 1]
```diff
@@ -115,11 +115,11 @@ def same_alias_target?(record)
     create
     Chef::Log.info "Record created: #{name}"
   elsif !same_record?(record)
-    unless overwrite == false
+    if overwrite
```
Comment (mi-wood) [inline providers/record.rb:118]: thank you!

**Comment by mi-wood** [comment]
can you bump the version in the metadata?

**Comment by krishnan-chandra** [comment]
Ping @mi-wood

**Comment by mi-wood** [comment]
Lgtm

**Pull request merged**